packages/discussions/src/types.ts

```diff
@@ -837,7 +837,7 @@ export interface IChannelAclPermission
     IWithEditor,
     IWithTimestamps {
   id: string;
-  restrictedBefore: Date;
+  restrictedBefore: string;
```

Comment:
When it comes out of DB as a field the result of a typeorm query, is it a string?  The DB column is a timestamp, so it's possible that it will be a Date object.

Reply:
Is the Date not converted to a string for the http response from the service?

Reply:
yeah, nvm.  What I was thinking of would be covered the ChannelAclEntity interface.

Reply:
👍